pkg/build/builder/sti.go

```diff
@@ -384,3 +393,13 @@ func scriptProxyConfig(build *api.Build) (*s2iapi.ProxyConfig, error) {
 	}
 	return config, nil
 }
+
+func copyArtifactSourceList(artifactsMapping []api.ImageSourcePath) (volumeList []s2iapi.VolumeSpec) {
```

Comment:
Do we really need to use named variable for returning here? I don't know what is the code style but I personally trying to not use them.